Insert the spaces between literals to satisfy clang with c++17 (or
even c++11) standard.

	-mi

--- generators/win32/borland_bmake.cpp	2007-02-02 14:01:28 UTC
+++ generators/win32/borland_bmake.cpp
@@ -383,7 +383,7 @@ BorlandMakefileGenerator::init()
     if(project->isEmpty("QMAKE_INSTALL_DIR"))
 	project->variables()["QMAKE_INSTALL_DIR"].append("$(COPY_DIR)");
 
-    bool is_qt = (project->first("TARGET") == "qt"QTDLL_POSTFIX || project->first("TARGET") == "qtmt"QTDLL_POSTFIX);
+    bool is_qt = (project->first("TARGET") == "qt" QTDLL_POSTFIX || project->first("TARGET") == "qtmt" QTDLL_POSTFIX);
     QStringList &configs = project->variables()["CONFIG"];
     if (project->isActiveConfig("shared"))
 	project->variables()["DEFINES"].append("QT_DLL");
--- generators/win32/mingw_make.cpp	2007-02-02 14:01:28 UTC
+++ generators/win32/mingw_make.cpp
@@ -394,7 +394,7 @@ MingwMakefileGenerator::init()
     if(project->isEmpty("QMAKE_INSTALL_DIR"))
 	project->variables()["QMAKE_INSTALL_DIR"].append("$(COPY_DIR)");
 
-    bool is_qt = (project->first("TARGET") == "qt"QTDLL_POSTFIX || project->first("TARGET") == "qt-mt"QTDLL_POSTFIX);
+    bool is_qt = (project->first("TARGET") == "qt" QTDLL_POSTFIX || project->first("TARGET") == "qt-mt" QTDLL_POSTFIX);
     project->variables()["QMAKE_ORIG_TARGET"] = project->variables()["TARGET"];
 
     // LIBS defined in Profile comes first for gcc
--- generators/win32/msvc_dsp.cpp	2007-02-02 14:01:27 UTC
+++ generators/win32/msvc_dsp.cpp
@@ -620,7 +620,7 @@ DspMakefileGenerator::init()
     if ( project->variables()["QMAKESPEC"].isEmpty() )
 	project->variables()["QMAKESPEC"].append( getenv("QMAKESPEC") );
 
-    bool is_qt = (project->first("TARGET") == "qt"QTDLL_POSTFIX || project->first("TARGET") == "qt-mt"QTDLL_POSTFIX);
+    bool is_qt = (project->first("TARGET") == "qt" QTDLL_POSTFIX || project->first("TARGET") == "qt-mt" QTDLL_POSTFIX);
     project->variables()["QMAKE_ORIG_TARGET"] = project->variables()["TARGET"];
 
     QStringList &configs = project->variables()["CONFIG"];
--- generators/win32/msvc_nmake.cpp	2007-02-02 14:01:28 UTC
+++ generators/win32/msvc_nmake.cpp
@@ -479,7 +479,7 @@ NmakeMakefileGenerator::init()
     if(project->isEmpty("QMAKE_INSTALL_DIR"))
 	project->variables()["QMAKE_INSTALL_DIR"].append("$(COPY_DIR)");
 
-    bool is_qt = (project->first("TARGET") == "qt"QTDLL_POSTFIX || project->first("TARGET") == "qt-mt"QTDLL_POSTFIX);
+    bool is_qt = (project->first("TARGET") == "qt" QTDLL_POSTFIX || project->first("TARGET") == "qt-mt" QTDLL_POSTFIX);
     project->variables()["QMAKE_ORIG_TARGET"] = project->variables()["TARGET"];
 
     QString targetfilename = project->variables()["TARGET"].first();
--- generators/win32/msvc_vcproj.cpp	2007-02-02 14:01:28 UTC
+++ generators/win32/msvc_vcproj.cpp
@@ -1094,8 +1094,8 @@ void VcprojGenerator::initOld()
 	project->variables()["QMAKESPEC"].append( getenv("QMAKESPEC") );
 
     bool is_qt =
-	( project->first("TARGET") == "qt"QTDLL_POSTFIX ||
-	  project->first("TARGET") == "qt-mt"QTDLL_POSTFIX );
+	( project->first("TARGET") == "qt" QTDLL_POSTFIX ||
+	  project->first("TARGET") == "qt-mt" QTDLL_POSTFIX );
 
    QStringList &configs = project->variables()["CONFIG"];
 
